Userland/Games/Spider/Game.cpp

```diff
@@ -284,30 +288,59 @@ void Game::mouseup_event(GUI::MouseEvent& event)
         return;
 
     bool rebound = true;
-    for (auto& stack : m_stacks) {
-        if (stack.is_focused())
-            continue;
+    if (event.button() == GUI::MouseButton::Secondary) {
+        // This enables the game to move the focused cards to the first possible stack excluding empty stacks.
+        // NOTE: I decided to ignore empty stacks, as the game has no undo button, and a card, which has been moved to an empty stack without any other possibilities is not reversable.
```

Comment:
We usually write notes as "This ignores empty stacks...", it'd be nice to keep that style here : )